lib/OLED/src/OLED.h

```diff
@@ -0,0 +1,14 @@
+#ifndef oled_h
+#define oled_h
+
+class OLED
+{
+    public:
+        bool init();
+        bool refresh();
+        bool printInt(int n);
```

Comment:
I probably didn't specify enough in the ticket so I will go back and refine the requirements, but in its current state this is not the most useful as it can only display one integer at a time. We will likely want it to be able to take in a string and add that to the display, but I will specify in the ticket once I think about it a bit. Sorry for the scope change :/